src/js/sdk.tsx

```diff
@@ -246,3 +246,14 @@ export function configureScope(callback: (scope: Scope) => void): ReturnType<Hub
   };
   getCurrentHub().configureScope(safeCallback);
 }
+
+/**
+ * Returns if the app crashed in the last run.
+ */
+export async function crashedLastRun(): Promise<boolean> {
```

Comment:
Overall everything looks good. I have only one comment.

All the fallbacks result to `false` which might be false positive, as we are just unable to read the value. To avoid that I think returning `boolean | undefined` is better, as users of the API can decide what to do if the value was not retrieved.  

```suggestion
export async function crashedLastRun(): Promise<boolean | undefined> {
```

Reply:
Also agree with it, returning false when we are not certain could give a false impression that the app didn't crash